crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1559,11 +1607,32 @@ where
                     non_retryable_errors: group_errors(state.non_retryable_errors),
                 }
             }
-        })
+        })?;
+
+        // Use best efforts to send the cert to remaining validators.
+        let metrics = self.metrics.clone();
+        spawn_monitored_task!(async move {
```

Comment:
Why does this help though? Under normal timing conditions, we should have been able to send requests to all validators before we receive 2f+1 responses, and each validator would spawn a thread to process the cert even if the connection closes.

Reply:
We think this is what happens today. But want to double verify. See discussions here https://mysten-labs.slack.com/archives/C055YL8CFBK/p1682884030792949?thread_ts=1682823380.568339&cid=C055YL8CFBK